src/wacks4shop/shift4shop.py

```diff
@@ -127,38 +127,60 @@ def determine_sales(self, timestamp: Optional[str]) -> list[Shift4ShopSale]:
         ordered_sales = sorted(sales, key=lambda sale: sale.datetime)
         return ordered_sales
 
-    def get_num_sales(self) -> int:
+    def _internal_get_num_sales(self, additional_query_filters) -> int:
         """
-        Get the current total number of sales by querying the Shift4Shop API.
+        Get the number of total sales using the shift4shop orders search API
         The API only supports filtering by 1 order status at a time so get the
         completed orders by taking the difference between all the orders without
         any filtering and the number of orders in the "not completed" status.
-        Additionally, there's a default "limit" applied (10) if you don't send that param
-        and an upper bound to how many orders can be returned at once so this is just
-        a stopgap solution.
-
-        Ultimately, we probably need to store previous num sales in the DB and use datestart
-        or invoicenumberstart in the future.
+        :param additional_query_filters:
+        See _request_orders() for filters that can be applied to the search query.
+        Examples include "limit", "orderstatus", "datestart"
+        :return:
```

Comment:
would prefer to not have this empty `:return:` in the docstring since this does return something. can either remove it, or put an int